auction-server/src/auction.rs

```diff
@@ -1249,21 +1265,20 @@ async fn verify_signatures_svm(
                     ),
                 })
                 .await;
-            opportunities.into_iter().any(|opportunity| {
-                let mut missing_signers = opportunity.get_missing_signers();
-                missing_signers.push(*relayer_pubkey);
-                all_signature_exists_svm(&message_bytes, accounts, &signatures, &missing_signers)
-            })
+            let opportunity = opportunities
```

Comment:
just a note: why do we check against existing opportunities? there are some tradeoffs to just simulating the tx and not checking that an opp exists for this PK, but that's how we handle on evm side, i don't recall why we decided on checking against existing opps for svm

Reply:
main reason here is to ignore the other necessary signature from wallet which comes at the end of the flow